### PR TITLE
Revert pr#3194 modification

### PR DESCRIPTION
**Pull request by lum1n0us**
- address values in call stack dump are relative to file beginning
- if running under fast-interp mode, address values are relative to every pre-compiled function beginning. not compatible with addr2line

### CODE REVIEW

**Comment by wenyongh** [comment]
@eloparco could you help check whether this PR is good to you? Thanks.

File: test-tools/addr2line/addr2line.py [hunk 1]
```diff
@@ -143,4 +189,14 @@ def main():
 
 
 if __name__ == "__main__":
+    print(
+        "**************************************************\n"
+        + "Before running this script, please make sure:\n"
+        + "  - the wasm file is compiled with debug info. (like: clang -g) \n"
+        + "  - the call-stack dump is generated by iwasm\n"
+        + "  - iwasm is compiled with -DWAMR_BUILD_DUMP_CALL_STACK=1\n"
+        + "  - iwasm isn't running under fast-interp mode. -DWAMR_FAST_INTERP=0\n"
```
Comment (eloparco) [inline test-tools/addr2line/addr2line.py:198]:
```suggestion
        + "  - iwasm isn't running under fast-interp mode. -DWAMR_BUILD_FAST_INTERP=0\n"
```

**Comment by eloparco** [comment]
>@eloparco could you help check whether this PR is good to you?

Tried it and it works! I was using fast interpreter mode, switching to to classic interpreter fixed it.

**Review by eloparco — APPROVED** [review]
LGTM with a minor comment on a typo

**Review by wenyongh — APPROVED** [review]

**Pull request merged**